src/SceneGraphDebugCommands.ts

```diff
@@ -0,0 +1,167 @@
+import * as vscode from 'vscode';
+import { SceneGraphCommandResponse, SceneGraphDebugCommandController } from 'roku-debug';
```

Comment:
These errors will be fixed once the version of roku-debug with this class has been published.